api/tests/unit/edge_api/identities/test_edge_identity_featurestate_view.py

```diff
@@ -0,0 +1,38 @@
+from django.urls import reverse
+from rest_framework import status
+
+
+def test_user_with_manage_identities_permission_can_retrieve_all_feature_states_for_identity(
```

Comment:
seems like it's the addition of the VIEW_ENVIRONMENT permission that's actually the one that's providing access here. Perhaps that's correct though?

Reply:
Ah yes, I have updated the test name